impeller/renderer/backend/gles/command_buffer_gles.cc

```diff
@@ -64,4 +64,13 @@ std::shared_ptr<BlitPass> CommandBufferGLES::OnCreateBlitPass() const {
   return pass;
 }
 
+// |CommandBuffer|
+std::shared_ptr<ComputePass> CommandBufferGLES::OnCreateComputePass() const {
+  if (!IsValid()) {
+    return nullptr;
+  }
+  // TODO(dnfield): implement compute for GLES.
```

Comment:
This isn't possible in GLES (anything < 3.2). So I think its best to just return nullptr with a comment here saying that compute passes aren't supported.

Reply:
Is the idea that >= 3.2 will just have Vulkan anyway?

I was assuming we could use extension methods to check if it's available and if so, use them.

Reply:
Yeah. Have a clear separation between a modern and legacy renderer with little effort made to update the legacy one.